src/main/java/com/github/checkstyle/regression/git/DiffParser.java

```diff
@@ -57,25 +57,36 @@ private DiffParser() {
      * @throws IOException     JGit library exception
      * @throws GitAPIException JGit library exception
      */
+    @SuppressWarnings("TryFinallyCanBeTryWithResources")
```

Comment:
I'm not sure what utility is generating this, but lets just turn it off instead of adding suppress warnings every time.
We will most likely never do this in this project as it is too hard to support code coverage.